markdown/org/docs/patterns/charlie/instructions/en.md

```diff
@@ -215,78 +226,94 @@ Getting it just right is one of those things you'll end up cherishing each time
 
-Place the zipper along the (slightly curved) edge of the fly extension.
+</tip>
 
-Now sew along the edge. No need to get close to the zipper, just sew along the edge to hold them in place.
+Edge stitch along the left edge of the zipper to the serged edge of the fly extension to hold it in place.
 
-### Attach zipper to the fly
+![Baste Zipper](BasteZipper.svg)
 
-Now place the right front leg's fly with the extra seam allowance that you folded back on top of the zip.
 
-Use a zipper foot edge-stitch the folded-over edge along the zipper edge.
+### Top stitch the right leg to the zipper
 
-<Tip>
+Start by pressing the right leg seam allowance to the back.
 
-You might want to baste this first to make sure it is nicely parallel to the zipper. Close, but not too close.
+<tip>
 
-</Tip>
+Remember that the seam allowance starts slightly right of the center front, the center front is marked in purple in the image, make sure you don't fold along the center front line by mistake.
+
+</tip>
 
-### Sew the zipper to the fly facing
+Ensure you place the folded edge of the right leg just left of the zipper teeth to prevent fabric from being caught in the zipper.
 
-Now sew the other side of the zipper to the fly facing. Make sure to align it properly so that
-when the trousers are closed, the zip is hidden.
+Stop sewing at the top 'fly extention' notch, slightly above the cross seam, and be sure to backstitch. If you don't do this you won't be able to keep the extension out of the way when sewing the J-Seam.
 
-### Finish the cross seam
+![Attach Extension to Right Leg](AttachExtensionToRightLeg.svg)
+
+
+### Attach the zipper to the fly facing.
+
+Be careful to align the leg left on the right center front. It can be valuable to pin or baste through the facing and right zide of the zipper tape. (We need to sew the left side of the zipper tape to the left leg).
+Ensuring everything is lined, sew the fly facing to the left side of the zipper. Try to get close to the zipper, but not too close.
+
+![Attach Facing to Zipper](AttachFacingToZipper.svg)
 
-Sew the last bit of the cross seam up to the highest fly notch.
 
 ### Sew the J-seam of the fly
 
-Now topstitch the so-called J-seam of the fly. Make sure to keep the fly extension out of the way.
+<tip>
+
+Use a peice of paper as a guide to sew the J seam, this can be cut from your pattern along the indicated stitch line.
 
-### Bar tack the bottom of the fly
+</tip>
 
-Now bartack the very botton of the J-seam (the horizontal part) but this time make sure to also catch the fly extension.
+Now topstitch the so-called J-seam of the fly. Make sure to keep the fly extension out of the way. Since you didn't sew all the way along the fly extention, you should be able to pin it out of the way.
```

Comment:
```suggestion
Now _topstitch_ the so-called J-seam of the fly. Make sure to keep the fly extension out of the way. Since you didn't sew all the way along the fly extention, you should be able to pin it out of the way.
```